receiver/podmanreceiver/podman_client.go

```diff
@@ -159,22 +160,28 @@ func (c *podmanClient) events() (chan Event, error) {
 	if err != nil {
 		return nil, err
 	}
+	maxRetries, err := strconv.Atoi(cfg.MaxRetries)
+	if err != nil {
+		return nil, err
```

Comment:
this conversion and possible error should be taken care of during config validation and struct should ideally have this defined as a number instead of string.

Reply:
You are right. But now this entire code should be removed because we will remove `max_retries` variable from the code as discussed on the slack.